packages/web-api/src/methods.ts

```diff
@@ -1593,25 +1641,174 @@ export interface ChatPostEphemeralArguments extends WebAPICallOptions, TokenOver
   icon_url?: string; // if specified, as_user must be false
   username?: string; // if specified, as_user must be false
 }
-export interface ChatPostMessageArguments extends WebAPICallOptions, TokenOverridable {
+
```

Comment:
This is the area of the code where I prototype modeling the `chat.postMessage` arguments differently, using unions and intersections instead of optional properties.